frontend/src/component/feature/FeatureStrategy/FeatureStrategyConstraints/AddSingleValueWidget.tsx

```diff
@@ -8,6 +8,7 @@ const StyledChip = styled(ValueChip, {
     shouldForwardProp: (prop) => prop !== 'hasValue',
 })<{ hasValue: boolean }>(({ theme, hasValue }) => ({
     color: hasValue ? 'inherit' : theme.palette.primary.main,
+    width: 'max-content',
```

Comment:
This is unrelated to the value reflow, but it prevents that chip from growing more than it needs to.